index.html

```diff
@@ -63,114 +63,114 @@ <h3 class="secondaryText">
+                    </ul>
+                    <button type="button" class="button">See Project</button>
+                </div>
             </div>
-        </div>
 
-        <div class="card workCard">
-            <div class="imagePlaceHolder"></div>
-            <div class="workInfo">
-                <h3  class="cardHeader">Multi-Post Stories <br>
-                    Gain + Glory</h3>
-                <ul class="tagsHandler workTags">
-                    <li>Ruby on Rails</li>
-                    <li>CSS</li>
-                    <li>JavaScript</li>
-                    <li>HTML</li>
-                </ul>
-                <button type="button" class="button">See Project</button>
+            <div class="card workCard">
+                <div class="imagePlaceHolder"></div>
+                <div class="workInfo">
+                    <h3  class="cardHeader">Multi-Post Stories <br>
+                        Gain + Glory</h3>
+                    <ul class="tagsHandler workTags">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="button">See Project</button>
+                </div>
             </div>
-        </div>
 
-        <div class="card workCard">
-            <div class="imagePlaceHolder"></div>
-            <div class="workInfo">
-                <h3  class="cardHeader">Multi-Post Stories <br>
-                    Gain + Glory</h3>
-                <ul class="tagsHandler workTags">
-                    <li>Ruby on Rails</li>
-                    <li>CSS</li>
-                    <li>JavaScript</li>
-                    <li>HTML</li>
-                </ul>
-                <button type="button" class="button">See Project</button>
+            <div class="card workCard">
+                <div class="imagePlaceHolder"></div>
+                <div class="workInfo">
+                    <h3  class="cardHeader">Multi-Post Stories <br>
+                        Gain + Glory</h3>
+                    <ul class="tagsHandler workTags">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="button">See Project</button>
+                </div>
             </div>
         </div>
-
     </section>
 
-    <section class="aboutme centeredGrid">
+    <section class="aboutme">
         <div class="sectionHeader aboutmeHeader">
             <h2>About me</h2>
             <h3> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
                 don’t hestiate to contact me.</h3>
             <button type="button" class="button">Get My Resume</button>
```

Comment:
- [ ] [OPTIONAL] It seems that this `<h3>` is not taking the full width as what we have in the Figma design below
![image](https://user-images.githubusercontent.com/55465133/130117316-4b3ff82f-f54d-45bf-925b-9ee6481a54ce.png)

- This is how yours is looking like
![image](https://user-images.githubusercontent.com/55465133/130117469-46548bb4-cefe-42a6-9160-b32093e76715.png)

You can give to the `<h3>` which was supposed to be a `<p>` tag a width of `60%` in the desktop version for example, to make it look like the original in Figma 👍 

- [ ] [OPTIONAL] You can also give the `margin-top` to the about me section since it is so close to the `my works` section 😉